.gitmodules

```diff
@@ -67,3 +67,7 @@
 [submodule "src/pybind/mgr/rook/rook-client-python"]
 	path = src/pybind/mgr/rook/rook-client-python
 	url = https://github.com/ceph/rook-client-python.git
+[submodule "src/aws-sdk-cpp"]
+	path = src/aws-sdk-cpp
+	url = https://github.com/prateek2211/aws-sdk-cpp
```

Comment:
Why not https://github.com/aws/aws-sdk-cpp ?

Reply:
had to tweak cmake a little bit and remove the modules which were not required by us, because aws-sdk-cpp is very huge,(even bigger than ceph entire codebase in terms of LoC)